scripts/framework_dev/__init__.py

```diff
@@ -0,0 +1,3 @@
+# This file needs to exist to make framework_dev a package.
```

Comment:
I don't think `framework_dev` is right as a package name. Unless that's intended to be a sub-package of some `mbedtls` package? The idea with calling the directory `mbedtls_dev` is that you can use it in a program that isn't specific to Mbed TLS, for example a program that puts multiple crypto libraries together and uses some code from `mbedtls_dev` to obtain information about things like supported crypto mechanisms. So the name should keep `mbedtls` in it.

The current name is `mbedtls_dev` because it's for development of or related to Mbed TLS, as opposed to `mbedtls` which should be reserved for a Python package that contains wrappers around Mbed TLS's C APIs. We could keep that name here. Or maybe change it to `mbedtls_framework`?

Reply:
I am happy with `mbedtls_dev` but I was assuming we'd want to generalise it given that it'll be used in TF-PSA-Crypto. I'll change this back to `mbedtls_dev`.